nametoaddr.c

```diff
@@ -126,7 +126,14 @@ pcap_nametoaddr(const char *name)
 	bpf_u_int32 **p;
 	struct hostent *hp;
 
+#ifdef _WIN32
+#pragma warning (push)
+#pragma warning (disable: 4996) /* disable MSVC's gethostbyname() warning here */
+	if ((hp = gethostbyname(name)) != NULL) {
+#pragma warning (pop)
```

Comment:
I'm OK with replacing ``disable: 4996`` with ``_WINSOCK_DEPRECATED_NO_WARNINGS``.

But I think ``#pragma warning (push)`` and ``#pragma warning (pop)`` are necessary because only this one statement needs to disable the warning. If we put the disabling code in a header, the whole following code will disable the warning, which is not what we need.

And why we need an ``#ifdef _MSC_VER``? The warning disabling doesn't work on all MSVCs?